.jenkins/pytorch/common_utils.sh

```diff
@@ -137,6 +137,44 @@ function install_triton() {
   fi
 }
 
+function setup_torchdeploy_deps(){
+  conda install -y cmake
+  conda install -y -c conda-forge libpython-static=3.10
+  sudo wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | sudo gpg --dearmor -o /usr/share/keyrings/magic-key.gpg && \
+  sudo echo "deb [arch=amd64,arm64 signed-by=/usr/share/keyrings/magic-key.gpg] https://apt.kitware.com/ubuntu/ bionic main" | sudo tee -a /etc/apt/sources.list && \
+  sudo echo "deb http://security.ubuntu.com/ubuntu focal-security main" | sudo tee -a /etc/apt/sources.list && \
+  sudo apt update && \
+  sudo apt install -y binutils && \
```

Comment:
after https://github.com/pytorch/multipy/pull/216 this shouldn't be required anymore

Reply:
will update